extensions/token_pg.go

```diff
@@ -73,13 +71,13 @@ func (t *TokenPG) configure(db interfaces.DB) error {
 }
 
 // HandleToken handles an invalid token
-func (t *TokenPG) HandleToken(token string) error {
+func (t *TokenPG) HandleToken(token string, game string, platform string) error {
 	l := t.Logger.WithFields(logrus.Fields{
 		"method": "HandleToken",
 		"token":  token,
 	})
 	l.Debug("deleting token")
-	query := fmt.Sprintf("DELETE FROM %s WHERE token = ?0;", t.tableName)
+	query := fmt.Sprintf("DELETE FROM %s WHERE token = ?0;", game+"_"+platform)
```

Comment:
I am not sure all games respect this rule =(

Reply:
omg, we need to be sure @guilhermef